### PR TITLE
Source rule helper

### PR DESCRIPTION
**Pull request by mconnell**
Adds a helper method for displaying a list of sources that are associated with a source rule.

This PR replaces PR #49 and corrects a history issue.

### CODE REVIEW

File: app/views/admin/rules/new.html.erb [hunk 1]
```diff
@@ -20,19 +20,9 @@
...
-            </td>
-            <td>
-              <ul class="table__cell-list">
-                <% source_rule.sources.each do |source| %>
-                  <li><%= source.name %></li>
-                <% end %>
-              </ul>
-            </td>
+            <td><%= check_box_tag 'applied_rule_ids[]', source_rule.id, true %></td>
+            <td><%= label_tag '', source_rule.name %></td>
```
Comment (benbarnett) [inline app/views/admin/rules/new.html.erb]: #lazyquestion Will this end up with the `for="id"` attribute for the checkbox above?

Comment (mconnell) [inline app/views/admin/rules/new.html.erb]: yerp

**Comment by benbarnett** [comment]
:+1:

**Comment by munckymagik** [comment]
:+1:

**Pull request merged**